rules-java-archives/addon/src/main/java/org/jboss/windup/rules/apps/java/archives/config/ArchiveIdentificationConfigLoadingRuleProvider.java

```diff
@@ -40,38 +41,71 @@ public ArchiveIdentificationConfigLoadingRuleProvider()
     }
 
     @Inject
-    private CompositeChecksumIdentifier identifier;
+    private CompositeArchiveIdentificationService identifier;
 
     @Override
     public Configuration getConfiguration(final GraphContext grCtx)
     {
-        ConfigurationBuilder config = ConfigurationBuilder.begin();
-        config.addRule().perform(new GraphOperation()
+        return ConfigurationBuilder.begin()
+                    .addRule()
+                    .perform(new AddSortedFileIndexOperation())
```

Comment:
Good point... I'll switch this to the in memory one. I would expect user-provided ones to be pretty small.